docs/Contributing/game-admin.rst

```diff
@@ -0,0 +1,170 @@
+  game tileset in the description.
+* Establish the length of turns. We have often run turns at 23 hours. However, lately we have been going with 25 hours.
+  Either way, make sure to mention it in the game description. If you intend to change the length of the turn at a
+  certain point in the game (e.g. for late game make the turns two days -- ``2*23`` or ``2*25``) you will definitely
+  want to state this up front.
+* Define up front how you want to handle RTS. RTS (Real Time Strategy) is a term we use to define how actions between
+  players can occur when both are playing at the same time. When a player RTS's, they will be acting against you at the
+  same time you are trying to do some moves, such as establishing a new city or attacking. A non-RTS game would not
+  allow this synchronous action. A RTS game would allow it. Teamless games are often non-RTS, same with Team games.
+  However, Experimental or other types are games could allow it.
+
+
+At this point you can announce the game on the ``#new-games`` channel on the Longturn Discord server to let people know
+about it. Ask a Discord Admin to create a channel for the game as well.
+
+While players are talking about the game on Discord and signing up on the website, you can get the game files ready. The
+easiest way to start is by copying a previous game and then editing the files. When finished you will combine the
+changes into a ``git commit`` and push them up to the games repository.
+
+.. code-block:: sh
+
+  ~/GitHub/games$ git status
+  ~/GitHub/games$ git add --all
+  ~/GitHub/games$ git commit
+  ~/GitHub/games$ git push origin
+
+
+The ``git status`` command gives you information on what changed on your local workspace. It gives you a chance to fix
+any files that may have changed that you did not intend to.  The ``git add`` command will add all changed files to the
+commit package.  The ``git commit`` command will bring up a text editor (often Nano) allowing you to enter in a commit
+message. Preface commit messages with the ``[Game Number]:``. Finally you push the changes up to the repo with the
+``git push`` command. You will be prompted for your SSH passkey to complete the step.
+
+
+Start the New Game
+==================
+
+When enough players have signed up, you will want to announce it on the game channel and set a potential start date.
+
+These are the steps to finalize a game:
+
+#. Go into the admin site on the Longturn server and set a start date at least 7 days into the future and save.
+#. Ask players to confirm participation on the website.
+#. Start editing the ``players.serv`` file and push changes up to the repo.
+#. Ask a server admin to setup a test game with the game server and map generator settings to see how things look.
+   :strong:`Note`: Game admins should test locally to check things out before asking a server admin to do it on the main
+   game server.
```

Comment:
Maybe add the following in an admonition:
> Server admins can perform additional actions when starting the game, such as looking at the map or performing edits. For example, it is common to post the size of the largest islands. Since this requires extra work, make sure you talk to the admin beforehand.